### ISSUE TITLE
0 - Ponerse de acuerdo con el code base

### ISSUE THREAD

**Issue by ivanchenoweth** [REUBATCODE/practica-html#1]
Hacer un concenso de las herramientas, lenguajes, entornos de desarrollo 

Comenten todos ...

**Comment by ivanchenoweth** [comment]
Yo sugiero usar codespaces para el ambiente en la nube, y localmente con alpine linux.
PHP + MySQL

**Comment by ivanchenoweth** [comment]
Como van a quedar la estructura de los archivos si no van a usar un framework?

**Comment by ivanchenoweth** [comment]
Necesito que comenten para que definan la estructura y el stack de tecnologia

**Comment by ivanchenoweth** [comment]
Canal de comuniacion

**Comment by ivanchenoweth** [comment]
Le voy a encargar a @REUBATCODE  que haga un proyecto de este repositorio para usar las tarjetas.
Tambien le voy a encargar que haga un milestone de el tiempo del proyecto en cuanto a planeacion y en cuanto a ejecucion

**Comment by ivanchenoweth** [comment]
- Recursos 
[Tutorial de PHP](https://www.w3schools.com/php/) en W3Schools
[Tutorial de MySQL](https://www.w3schools.com/sql/) en W3Schools
[Tutorial de PHP para principiantes](https://www.tutorialspoint.com/php/) en Tutorials Point
[Tutorial de MySQL para principiantes](https://www.tutorialspoint.com/mysql/) en Tutorials Point

**Comment by ivanchenoweth** [comment]
https://github.com/kunalbagnial/PHP-MySQL-Login-System

**Comment by REUBATCODE** [comment]
Yo propongo utilizar HTML, CSS y Javascript para nuestro proyecto de integradora.

**Comment by AngelDabnee** [comment]
Estoy de acuerdo con mi compañero Rubén, dado que hay documentación donde apoyarnos para poder realizar investigaciones y aplicarlas.

**Comment by ivanchenoweth** [comment]
> Le voy a encargar a @REUBATCODE que haga un proyecto de este repositorio para usar las tarjetas. Tambien le voy a encargar que haga un milestone de el tiempo del proyecto en cuanto a planeacion y en cuanto a ejecucion

No han atendido esto ...

**Comment by ivanchenoweth** [comment]
> Canal de comuniacion

Cuando se van a reunir para debatir o trabajar en esto? Debe ser diario compañeros

**Comment by AngelDabnee** [comment]
> > Canal de comuniacion
> 
> 
> 
> Cuando se van a reunir para debatir o trabajar en esto? Debe ser diario compañeros

Lo hemos estado realizando de manera diaria. Ahorita tenemos un conflicto sobre lo que le está pidiendo a Rubén ya que no encontramos cómo realizarlo. Pero tenemos además un grupo de WhatsApp's donde nos estamos poniendo de acuerdo

**Comment by REUBATCODE** [comment]
Esta historia será terminada cuando las páginas estén nombradas, los code bases bien definidos, y las estructuras hechas a mano estén revisadas.

**Comment by ivanchenoweth** [comment]
Cual va a ser la nomenclatura de sus ramas

**Comment by ivanchenoweth** [comment]
Cuando lo van a terminar

**Comment by ivanchenoweth** [comment]
Las nomenclatura de sus ramas, nunca las definieron.
Sugiermos que las ramas se van a llamar:
ft/no-rama

Ejemplo, del issue 10 - login
La rama debera llamarse:
ft/10

**Comment by JEstebanSanti** [comment]
- favor de revisar el documento 
https://docs.google.com/document/d/1YZRgpEKgikvVcO4-hmUcDYm5EXPMmlDZxbCBkvjYwPs/edit?usp=sharing

**Comment by ivanchenoweth** [comment]
arrreglar bugs ... bug/No-issue
